boards/nrf52dk/Makefile.include

```diff
@@ -14,7 +14,7 @@ export JLINK_DEVICE := nrf52
 
 # special options when using SoftDevice
 ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
-export JLINK_PRE_FLASH := loadfile $(BINDIR)/softdevice.hex
+export JLINK_PRE_FLASH := erase\nloadfile $(BINDIR)/softdevice.hex
```

Comment:
another solution is to use /bin/echo instead of echo like it is done some lines above in dist/tools/jlink/jlink.sh.
/bin/echo behaves as the bash version, so we will need to call it with -e .
Let me if you have a preference.
For portability I would better move to printf but for linux based system /bin/echo should work.